index.html

```diff
@@ -253,10 +253,10 @@ <h2>About<br>Myself</h2>
                     <h3>LET'S CONNECT</h3>
```

Comment:
- [ ] Kindly rectify the color contrast issue in this text
![lets-connect-text-color-access](https://user-images.githubusercontent.com/44057863/167712846-a18b93a3-068b-4d33-8802-6dd34e6d48ce.png)
.